packages/core/tests/ssr-only.test.ts

```diff
@@ -31,3 +31,23 @@ describe('createCss: SSR', () => {
     expect(styles).toEqual(secondStyles);
   });
 });
+
+// https://github.com/modulz/stitches/issues/200
+describe('#200 - getStyles() does not close keyframes block', () => {
```

Comment:
I dont think we need a new test for this

Instead, can we check the existing tests here: https://github.com/modulz/stitches/blob/canary/packages/core/tests/index.test.ts#L894

And a few other tests after that one too

The tests pass because the assertion is wrong, it's also missing the closing curly 🤔

Reply:
Oh, there is an existing one.

Reply:
There are quite a few tests that check for the generated CSS of `@keyframes` and they're all missing the bracket 😄